edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -42,18 +41,21 @@ public class KubernetesModule : Module
         readonly string deviceNamespace;
         readonly Uri managementUri;
         readonly Uri workloadUri;
-        readonly IEnumerable<AuthConfig> dockerAuthConfig;
+        readonly IEnumerable<global::Docker.DotNet.Models.AuthConfig> dockerAuthConfig;
```

Comment:
I think if you wind up using this more than once, it should be a `using` statement.

Reply:
Yeah, it is only for one place so far.